src/lib.rs

```diff
@@ -50,6 +51,7 @@ extern crate hyper;
 #[cfg(unix)]
 extern crate nix;
 extern crate alloc;
+extern crate chrono;
```

Comment:
can we use this crate to replace time crate later.
time crate is deprecated now.